artifacttemplates/http%3A%2F%2Fopentosca.org%2Fartifacttemplates/Tomcat_7-Configure-w1/README.md

```diff
@@ -1,4 +1,4 @@
-# Tomcat7_Impl_ConfigureIA_AT [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+# Tomcat7-Implementation_ConfigureIA_AT [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
"Implementation" should be in NodeTypeImpls but not in ArtifactTemplates regarding the new convention, right?